tests/Jenkins/dirac_ci.sh

```diff
@@ -135,6 +135,10 @@ function installSite(){
     return
   fi
 
+  
+  #replace the sources with custom ones if defined
+  diracReplace
```

Comment:
Using dirac-install++ (so from v6r21) this function is not needed anymore, instead one can use "-m" option of dirac-install

Reply:
I just moved it from bellow. we can change it later maybe, since this has nothing to do with M2Crypto